lib/spack/spack/paths.py

```diff
@@ -58,3 +59,16 @@
 mock_gpg_data_path = os.path.join(var_path, "gpg.mock", "data")
 mock_gpg_keys_path = os.path.join(var_path, "gpg.mock", "keys")
 gpg_path           = os.path.join(opt_path, "spack", "gpg")
+
+
+def is_package_file(filename):
```

Comment:
I think this might be more appropriate as a function in `repo.py` (which dictates how repositories are laid out) - they include a `repo.yaml` file.

Reply:
Moved.